CellFabric/Makefile

```diff
@@ -0,0 +1,23 @@
+#
+
+INPUT=/dataVolume/topology_output/Results
+
+LOC_GENERATOR=Cell_Fabric_FinFET__Mock
+CELL_GENERATOR=/CellFabric/$(LOC_GENERATOR)
+# Run the generator script
+
+${DESIGN}.map:	${DESIGN}.lef
+	ls *.gds.json -l | awk '{print $$NF}' | awk -F'.' '{print $$1,$$1".gds"}' > ${DESIGN}.map
+
+${DESIGN}.lef:	${INPUT}/${DESIGN}_lef.sh  ../PDK_Abstraction $(LOC_GENERATOR)
+	chmod u+x $<
+	(cd ${LOC_GENERATOR}; $< python3; cat *.lef > ${DESIGN}.lef)
```

Comment:
We need something here to make sure that we don't generate 3GB combined LEF files with many duplicate entries. (Probably removing all LEF files before we generate new ones.)